script/Arcueid/Day 1/01_00_ARC01_2.txt

```diff
@@ -492,12 +490,12 @@ If I recall correctly―――
 // In EN, the news tends to name the killer not the killings.
 // Modulated slightly here to match
 "What were they calling him again?
- I know it sounds like a really vulgar sales pitch.
+ Sounded like some kind of vulgar sales slogan...
```

Comment:
I thought 'vulgar tabloid headline' was a better suiting for this cos it's very idk The Sun to come up with a catchy killer name like that.